static/css/map-around.css

```diff
@@ -0,0 +1,40 @@
+/* marker */
+.marker {
+  fill: #297ddb;
+}
+
+.not-current-marker {
```

Comment:
Can't you use `.marker:not(.current-marker)` or something?

Reply:
Not sure what this does however. Can't use these style in `.marker` neither?

Reply:
Look better with you version for sure. It's an old style that I didn't address here yet. I want scope the map style on a specific PR focusing on it. This chage could be made after to not block this one?